internal/k8s/exec.go

```diff
@@ -62,10 +65,19 @@ func (c *Client) execInPod(ctx context.Context, p ExecParameters) (*ExecResult,
 	result := &ExecResult{}
 
 	err = exec.Stream(remotecommand.StreamOptions{
-		Stdin:  nil,
+		Stdin:  stdin,
 		Stdout: &result.Stdout,
 		Stderr: &result.Stderr,
-		Tty:    false,
+		Tty:    true,
 	})
+
+	// Graceful close of stdin (no Ctrl-C sent)
+	stdin.Close()
```

Comment:
A `defer stdin.Close()` right after creation might be better, `stdin` will leak when the error return above executes.

Reply:
Done, even though there were no more error returns in between.